README.md

```diff
@@ -1,10 +1,127 @@
-## CREATE AN ASSIGNMENT ON CANVAS.
+# CodEval
+
+Currently CodEval has 3 main components:
+## 1. Test Simple I/O Programming Assignments on Canvas
+### codeval.ini contents
+```
+[SERVER]
+url=<canvas API>
+token=<canvas token>
+[RUN]
+precommand=
+command=
+```
+### Command to run:  
+`python3 codeval.py <a unique part of assignment name> [FLAGS]`  
+Example:  
+If the course name on Canvas is CS 149 - Operating Systems, the command can be:  
+`python3 codeval.py CS\ 149`  

```

Comment:
i think there is a command parameter now. also, i usually run the pyz file rather than the codeval.py file. (it's easier to set up in cron.)